pkg/services/store/kind/dashboard/testdata/gdev-walk-graph-gradient-area-fills.json

```diff
@@ -15,15 +15,17 @@
       "kind": "panel",
       "name": "Req/s",
       "URL": "/d/graph-gradient-area-fills.json/panel-tests-graph-gradient-area-fills?viewPanel=2",
+      "fields": {
+        "type": "graph"
+      },
       "references": [
         {
-          "kind": "ds",
-          "type": "default.type",
```

Comment:
@ArturWierzbicki -- NOTE the version used in bluge will still fill in the default values.